docs/schema.md

```diff
@@ -1,22 +1,22 @@
-# Extracted Record Schema
+# Extracted record schema
 
-Algolia is a schema-less search engine. However to provide a simple and easy experience with Neltify and the Crawler,
-we currently populate Algolia with a standard record.
+Algolia is a schemaless search engine. However to provide an effortless experience with Netlify and the Algolia Crawler, the plugin currently populates your Algolia index with a standard record schema.
 
-All root level properties are a computation of multiple selectors with fallback. We **might change the extraction logic** or to add more properties, but **will not remove root level properties without a proper deprecation period**.
-All properties marked as "not undefined" will be present in the end record, the other can be missing if did not find any relevant information.
+All root-level properties are a computation of multiple selectors, with a fallback. We might change the extraction logic to add more properties, but **we won't remove root-level properties without a proper deprecation period**.
+
+All properties that aren't marked as optional are present in the final record. Others can be missing if the Algolia Crawler coudln't find any relevant information.
 
 ## Schema
 
 ```ts
 {
   /**
-   * The Algolia unique identifier
+   * The object's unique identifier
    */
   objectID: string;
 
   /**
-   * Where did we found this record
+   * The URL where the Algolia Crawler found the record
```

Comment:
Maybe it's worth adding a note on canonical URLs? I can definitely see it becoming an issue if users don't understand it.

Reply:
mmh indeed, but that's more an internal of the crawler than specific to the plugin.
not sure it's the best place